opentelemetry/daemonset/README.md

```diff
@@ -0,0 +1,22 @@
+# DaemonSet
+
+Overall, this configuration sets up an OpenTelemetry collector deployed as daemonset that receives Jaeger traces, adds Kubernetes attributes, and exports the traces to Tempo over OTLP.
+
+## How to run
+1. Create the `SecurityContextConstraints`: `oc apply -f scc.yaml`
```

Comment:
Will it be more readable if we put all the commands on extra lines?

Like this?
```suggestion
1. Create the `SecurityContextConstraints`: 
<code>
oc apply -f scc.yaml
</code>
```

Code would have to be replaced with ```. Unfortunately I don't know how to escape this for a suggestion. :raccoon: